modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -207,7 +207,10 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
       case engine.GetState(_)            => NullEvent
       case engine.GetSeqState(_, _)      => NullEvent
       case engine.ActionStop(_, _)       => ActionStopRequested(svs)
-      case engine.Log(msg)               => NewLogMessage(msg)
+      case engine.LogDebug(msg)          => NewLogMessage(msg)
```

Comment:
This is not needed. We should send them to `NullEvent`. How the log works is that we capture log events at the logger level using log4s/slf4j and those are sent to the client as events of type `ServerLogMessage`
In fact we may remove `NewLogMessage` altogether